src/TimePicker.jsx

```diff
@@ -270,12 +285,15 @@ TimePicker.propTypes = {
   disableClock: PropTypes.bool,
   disabled: PropTypes.bool,
   format: PropTypes.string,
+  hourAriaLabel: PropTypes.string,
   isOpen: PropTypes.bool,
   locale: PropTypes.string,
   maxDetail: PropTypes.oneOf(allViews),
   maxTime: isTime,
+  minAriaLabel: PropTypes.string,
```

Comment:
I'd suggest

```suggestion
  minuteAriaLabel: PropTypes.string,
```

since `min` may be confusing. Minute? Minimum?